### PR TITLE
[CoreLocation] Add support for Xcode 13 beta 4.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/corelocation.cs [hunk 1]
```diff
@@ -134,6 +134,10 @@ partial interface CLLocation : NSSecureCoding, NSCopying, CKRecordValue {
 		[Export ("initWithCoordinate:altitude:horizontalAccuracy:verticalAccuracy:course:courseAccuracy:speed:speedAccuracy:timestamp:")]
 		IntPtr Constructor (CLLocationCoordinate2D coordinate, double altitude, double hAccuracy, double vAccuracy, double course, double courseAccuracy, double speed, double speedAccuracy, NSDate timestamp);
 
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("initWithCoordinate:altitude:horizontalAccuracy:verticalAccuracy:course:courseAccuracy:speed:speedAccuracy:timestamp:sourceInfo:")]
+		IntPtr Constructor (CLLocationCoordinate2D coordinate, double altitude, double hAccuracy, double vAccuracy, double course, double courseAccuracy, double speed, double speedAccuracy, NSDate timestamp, CLLocationSourceInformation sourceInfo);
```
Comment (spouliot) [inline src/corelocation.cs:139]: minor: would be nicer to use `horizontalAccuracy` and `verticalAccuracy` like the selector does

**Review by spouliot — APPROVED** [review]

File: src/corelocation.cs (hunk 1 above)
Comment (mandel-macaque) [inline src/corelocation.cs:139]:
```suggestion
		IntPtr Constructor (CLLocationCoordinate2D coordinate, double altitude, double horizontalAccuracy, double verticalAccuracy, double course, double courseAccuracy, double speed, double speedAccuracy, NSDate timestamp, CLLocationSourceInformation sourceInfo);
```

**Review by dalexsoto — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Unrelated issues https://github.com/xamarin/maccore/issues/2481

**Pull request merged**